antismash/detection/hmm_detection/data/hmmdetails.txt

```diff
@@ -151,6 +151,25 @@ PF07366	SnoaL family	20	PF07366.hmm
 thio_amide	nosA tpdK nocA berI pbtH	20	thio_amide.hmm
 phzB	Phenazine biosynthesis gene	80	phzB.hmm
 phosphonates	Phosphonate biosynthesis PEP mutase	275	phosphonates.hmm
+phosphonatesV2	Phosphonate biosynthesis PEP mutase	150	phosphonatesV2.hmm
+phosphonatesHydrolase	Phosphonopyruvate hydrolase	400	phosphonatesHydrolase.hmm
+phosphonatesMeIsocitrateLyase	Methylisocitrate Lyase	250	phosphonatesMeIsocitrateLyase.hmm
+2-Hacid_dh_C	D-isomer specific 2-hydroxyacid dehydrogenase, NAD binding domain	25	2-Hacid_dh_C.hmm
+Aldedh	Aldehyde dehydrogenase family	23	Aldedh.hmm
+Aminotran_1_2	Aminotransferase class I and II	19	Aminotran_1_2.hmm
+Aminotran_5	Aminotransferase class-V	21	Aminotran_5.hmm
+AurF	P-aminobenzoate N-oxygenase AurF	29	AurF.hmm
+CTP_transf_like	Cytidylyltransferase-like	21	CTP_transf_like.hmm
+DUF4992	Domain of unknown function	29	DUF4992.hmm
+Fe-ADH	Iron-containing alcohol dehydrogenase	26	Fe-ADH.hmm
+HMGL-like	HMGL-like	29	HMGL-like.hmm
```

Comment:
This is a duplicate name of `HGML-like` earlier in the file, which is the same Pfam (though a slightly older version). It would be better to update that file than have two copies, but we'll need to check it doesn't impact any other rule using it (currently only `betalactone`).

Reply:
Resolved by removing the duplicate HMGL-like as it is still detected using the previous model